### PR TITLE
Revised Team Addition to index.html

### PR DESCRIPTION
**Pull request by joelbitar1986**
* Fixed responsive positioning issues
* Removed line between divs on mobile by adding viewport meta tag
* Fixed social link URLS
* Cleaned up CSS
* Made photos smaller

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,7 +44,7 @@
       </div>
 
       <div class="row">
-        <form id="contact" method="post">
```
Comment (steveafrost) [inline index.html]: thanks!

File: index.html [hunk 2]
```diff
@@ -63,10 +65,44 @@
...
-        </form>
+         </form>
+        </div>
       </div>
     </div>
+
+    <div id="team">
+        <div class="container">
+          <div class="row">
+            <div>
```
Comment (steveafrost) [inline index.html]: Why is this extra div with no class needed again? I took it out and didn't notice the difference.

File: stylesheets/main.css [hunk 3]
```diff
@@ -1,54 +1,94 @@
...
-  color: #666666;
+#contact {
+  background: -moz-linear-gradient(90deg, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 4%, rgba(238,99,82,1) 19%); /* ff3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(81%, rgba(238,99,82,1)), color-stop(96%, rgba(150,114,114,1)), color-stop(100%, rgba(150,114,114,1)), color-stop(100%, rgba(150,114,114,1))); /* safari4+,chrome */
+  background: -webkit-linear-gradient(90deg, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 4%, rgba(238,99,82,1) 19%); /* safari5.1+,chrome10+ */
+  background: -o-linear-gradient(90deg, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 4%, rgba(238,99,82,1) 19%); /* opera 11.10+ */
+  background: -ms-linear-gradient(90deg, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 4%, rgba(238,99,82,1) 19%); /* ie10+ */
+  background: linear-gradient(0deg, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 0%, rgba(150,114,114,1) 4%, rgba(238,99,82,1) 19%); /* w3c */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#EE6352', endColorstr='#967272',GradientType=0 ); /* ie6-9 */
+  padding-top: 80px;
```
Comment (steveafrost) [inline stylesheets/main.css]: Can we add padding, especially padding top, using em or vw? The 80px is a big space on mobile.

File: stylesheets/main.css [hunk 4]
```diff
@@ -1,54 +1,94 @@
...
   background-color: #f58a6c;
   border-color: #f58a6c;
 }
 
 h1 {
-  font-size: 4em;
-  margin-bottom: 0.25em;
-  margin-top: 1em;
+    font-size: 5em;
+    margin-bottom: 0.25em;
```
Comment (steveafrost) [inline stylesheets/main.css]: Can we keep the margins off the general classes like h1, h2, p, span, etc? Might not want that spacing everything is what I'm thinking. Your thoughts?

Comment (joelbitar1986) [inline stylesheets/main.css]: Definitely, I think that was from your initial styling.

File: index.html (hunk 2 above)
Comment (joelbitar1986) [inline index.html]: I removed it.

File: stylesheets/main.css (hunk 3 above)
Comment (joelbitar1986) [inline stylesheets/main.css]: Used vh instead of px

**Comment by joelbitar1986** [comment]
My current issues:

General: 
*Not a big fan of the new 'send' button. The previous one was much sexier.

Desktop: 

*Not enough space at the top of the page before 'Spitball'. My personal preference (subjective) is to have page contents more vertically centered on page load.
*Too much space below '#team' div. There is too much room to scroll to get the bottom. 
* Something strange about the way our pictures look at the moment. They are either too far apart or too small. 

Mobile:
* Need some padding at the bottom of the page below team section (glyphicons are nearly touching bottom of the page)
* 'The Team' is too close to send button on iphone sized screens. There needs to be some margin or padding between contact and team sections.

**Comment by joelbitar1986** [comment]
Desktop: 

* Not enough space at the top of the page before 'Spitball'. It looks good on mobile because there is more real estate being utilized but strange when there is empty space on the page. My personal preference (subjective) is to have page contents more vertically centered on page load.

screenshot from 2017-05-25 00-37-32

**Comment by joelbitar1986** [comment]
Something strange about the way our pictures look at the moment. They are either too far apart or too small. 

screenshot from 2017-05-25 00-44-06

**Comment by joelbitar1986** [comment]
'The Team' is too close to send button on iphone sized screens. There needs to be some margin or padding between contact and team sections.

screenshot from 2017-05-25 00-46-58

**Comment by joelbitar1986** [comment]
Need some padding at the bottom of the page below team section (glyphicons are nearly touching bottom of the page)

screenshot from 2017-05-25 00-51-45

**Comment by joelbitar1986** [comment]
Before we merge: what were your thoughts on my comments?

**Comment by steveafrost** [comment]
Totally missed them. Yeah spacing needs to be fixed on mobile most. The 100vh can probably be taken out now that we're not doing divs by color and that will help fix that. Without 100vh in there, we can add padding based on vh. I'm OK about the padding on the top for now but agree with all other spacing comments.

Big screen real estate being wasted: we can do some drawings on the side for larger screens.

**Pull request merged**